conftest.py

```diff
@@ -249,8 +255,9 @@ def hashing(request):
     # Either save or compare
     utils._handle_hashes(answer_dir, request.cls.answer_file, hashes,
         request.config.getoption('--answer-store'))
-    if request.config.getoption('--save-answer-arrays'):
+    if request.config.getoption('--answer-raw-arrays'):
         # answer_file has .yaml appended to it, but here we're saving
         # the arrays as .npy files, so we remove the .yaml extension
-        utils._save_arrays(answer_dir, request.cls.answer_file.split('.')[0],
-            request.cls.hashes, request.config.getoption('--answer-store'))
+        utils._handle_raw_arrays(array_dir, request.cls.answer_file.split('.')[0],
```

Comment:
could you expand this function call with explicit keyword arguments syntax ? with that many arguments it's a bit hard to evaluate which one I should check first without knowing what their respective roles are supposed to be.